src/components/PageBlocks/CalendarBase.vue

```diff
@@ -143,6 +143,8 @@ export default {
         editable: false,
         eventLimit: true,
         locale: this.locale,
+        // prevents showing info of prev and next month
+        showNonCurrentDates: false,
```

Comment:
Not quite unfortunately, we dont want to hide them. Just use those dates to fetch events